Makefile

```diff
@@ -0,0 +1,30 @@
+.DEFAULT_GOAL := install
+
+.PHONY: check
+check:
+	@echo Checking...
+	@git diff -s --exit-code . || (echo "Build failed: one of more files aren't staged to be committed" && exit 1)
```

Comment:
Could we simply show the diff and not suppress its errors?

Reply:
The diff would be huge, as it will only happen when the generated manifests aren't updated. We have a similar target in the Jaeger Operator, and when it fails, we know it's because the generated files aren't up to date.

How about we start with this silent approach, and change it to a verbose mode if we need?

Reply:
We have had these cases and I like seeing the diff on CI, as it allows me to pin point what's actually wrong on CI.
It should be too big normally: https://cloud.drone.io/observatorium/configuration/135/1/2

Reply:
I'll move this to another PR. For now, I just removed it from here.